webhooks/grading/grading.js

```diff
@@ -30,8 +34,24 @@ function processResults(data) {
 router.post('/', function(req, res, next) {
 
     const data = req.body;
+    if (data.event === 'grading_started') {
+        if (data.job_id === undefined || data.job_id === null || !Number.isInteger(data.job_id)) {
```

Comment:
Ah, didn't know `isNumber()` would handle those, that's a much cleaner solution.